tests/std/test.lst

```diff
@@ -278,6 +278,8 @@ tests\P0896R4_ranges_range_machinery
 tests\P0896R4_ranges_subrange
 tests\P0896R4_ranges_test_machinery
 tests\P0896R4_ranges_to_address
+tests\P0896R4_ranges_alg_unique
+tests\P0896R4_ranges_alg_unique_copy

```

Comment:
These should appear in sorted order.

Reply:
Give me a second to find a suitable ordering that does not necessitate changes here